packages/animations/lib/src/page_transition_switcher.dart

```diff
@@ -195,7 +188,7 @@ class PageTransitionSwitcher extends StatefulWidget {
   ///
   /// The child is considered to be "new" if it has a different type or [Key]
   /// (see [Widget.canUpdate]).
-  final Widget child;
+  final Widget? child;
```

Comment:
Should there be a ? for child ? Can this class have a null child ?

Reply:
There should be a `?` for child. This class can have a null child, a case which is handled on L307:
https://github.com/flutter/packages/blob/master/packages/animations/lib/src/page_transition_switcher.dart#L307